migrations/2021-06-15-151515_circulating_supply_table/up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE circulating_supply
+(
+    block_timestamp          numeric(20, 0) NOT NULL,
+    block_hash               text           NOT NULL,
+    value                    numeric(45, 0) NOT NULL,
+    total_supply             numeric(45, 0) NOT NULL,
+    lockups_number           numeric(45, 0) NOT NULL,
+    active_lockups_number    numeric(45, 0) NOT NULL,
+    foundation_locked_supply numeric(45, 0) NOT NULL,
+    lockups_locked_supply    numeric(45, 0) NOT NULL
+);
+
+ALTER TABLE ONLY circulating_supply
+    ADD CONSTRAINT circulating_supply_pkey PRIMARY KEY (block_hash);
+
+CREATE INDEX circulating_supply_timestamp_idx ON circulating_supply USING btree (block_timestamp);
+
+ALTER TABLE ONLY circulating_supply
+    ADD CONSTRAINT circulating_supply_fk FOREIGN KEY (block_hash) REFERENCES blocks (block_hash) ON DELETE CASCADE;
+
+CREATE VIEW lockups AS
```

Comment:
Apply the same reasoning for the naming as one I listed above, so we don't confuse people with having `accounts` and `lockups` on the same flat level.